xbmc/settings/Settings.h

```diff
@@ -123,6 +123,7 @@ class CSettings : public CSettingsBase, public CSettingCreator, public CSettingC
   static const std::string SETTING_VIDEOPLAYER_USEVAAPIVC1;
   static const std::string SETTING_VIDEOPLAYER_USEVAAPIVP8;
   static const std::string SETTING_VIDEOPLAYER_USEVAAPIVP9;
+  static const std::string SETTING_VIDEOPLAYER_USEVAAPIHEVC;
```

Comment:
lets's start dropping this useless consts. add the string directly to vaapi

Reply:
Why are they useless? They prevent typos

Reply:
True